packages/rocketchat-slashcommands-leave/leave.js

```diff
@@ -0,0 +1,31 @@
+
+/*
+ * Leave is a named function that will replace /leave commands
+ * @param {Object} message - The message object
+ */
+	if (Meteor.isClient) {
+		RocketChat.slashCommands.add('leave', null, {
```

Comment:
To me `undefined` reads and is more "javascripty" than `null`, so I would personally prefer `undefined`.